pkg/driver/opcua/driver.go

```diff
@@ -49,10 +50,9 @@ type Driver struct {
 
 func (d *Driver) applyConfig(ctx context.Context, cfg config.Root) error {
 
-	config.SetDefaults(&cfg)
-	a, undo := node.AnnounceScope(d.announcer)
+	a := node.NewReplaceAnnouncer(d.announcer).Replace(ctx)
```

Comment:
This line needs to be in the `Factory.New` method and a saved field stored in `*Driver`